facefusion/face_analyser.py

```diff
@@ -164,64 +164,42 @@ def detect_with_retinaface(temp_frame : Frame, temp_frame_height : int, temp_fra
 	return bbox_list, kps_list, score_list
 
 
-def detect_with_yoloface(temp_frame : Frame, face_detector_height : int, face_detector_width : int) -> Tuple[List[Bbox], List[Kps], List[Score]]:
+def detect_with_yoloface(temp_frame: Frame, face_detector_height: int, face_detector_width: int) -> Tuple[List[Bbox], List[Kps], List[Score]]:
 	face_detector = get_face_analyser().get('face_detector')
-	input_size = (face_detector_height, face_detector_width)
+	bbox_list = []
+	kps_list = []
+	score_list = []
 	shape = temp_frame.shape[:2]
-	ratio = min(input_size[0] / shape[0], input_size[1] / shape[1])
-	new_unpad = int(round(shape[1] * ratio)), int(round(shape[0] * ratio))
-	dw, dh = (input_size[1] - new_unpad[0]) / 2, (input_size[0] - new_unpad[1]) / 2
+	ratio = min(face_detector_height / shape[0], face_detector_width / shape[1])
```

Comment:
we cannot use the ratio_height / ratio_width that are being passed like in retinaface?